lib/msal-core/test/Configuration.spec.ts

```diff
@@ -118,4 +118,31 @@ describe("Configuration.ts Type", () => {
         const { framework } = configWithDefaults;
         expect(framework.isAngular).to.eq(fake_isAngular);
     });
+    it("does not set defaults for telemetry object", () => {
+        const config: Configuration = {
+            auth: {
+                clientId: "iamnotreal"
+            }
+        };
+        const configWithDefaults: Configuration = buildConfiguration(config);
+        const { system } = configWithDefaults;
+        expect(system.telemetry).to.be.undefined;
+    });
+    it("does not set defaults for telemetry object memebers", () => {
+        const config: Configuration = {
+            auth: {
+                clientId: "iamnotreal"
```

Comment:
Can we use string constant variables instead of hardcoding strings in tests?

Reply:
updating